controllers/grid/navigationMenus/NavigationMenusGridCellProvider.inc.php

```diff
@@ -53,10 +53,28 @@ function getTemplateVarsFromRowColumn($row, $column) {
 		assert(is_a($navigationMenu, 'NavigationMenu') && !empty($columnId));
 
 		switch ($columnId) {
-		    case 'title':
-		        return array('label' => '');
-		    default:
-		        break;
+			case 'title':
+				return array('label' => '');
+			case 'nmis':
+				$navigationMenuItemDao = DAORegistry::getDAO('NavigationMenuItemDAO');
+				$items = $navigationMenuItemDao->getByMenuId($navigationMenu->getId())->toArray();
+
+				$navigationMenusTitles = '';
+
+				$templateMgr = TemplateManager::getManager(Application::getRequest());
+				import('classes.core.ServicesContainer');
+				foreach ($items as $item) {
+					ServicesContainer::instance()
+						->get('navigationMenu')
+						->transformNavMenuItemTitle($templateMgr, $item);
+					$navigationMenusTitles = $navigationMenusTitles.$item->getLocalizedTitle().', ';
+				}
+
+				$navigationMenusTitles = trim($navigationMenusTitles, ', ');
+
+				return array('label' => $navigationMenusTitles);
```

Comment:
It's fine how you build the string here. As a suggestion, I find I can sometimes save some string management headaches by joining an array instead. Quick example:

```
$navigationMenuItems = array();
foreach($items as $item) {
  $navigationMenuItems[] = trim($title);
}
return join(', ', $navigationMenuItems);
```

Also, as part of [another PR](https://github.com/pkp/pkp-lib/pull/3341), I'm introducing a `common.listSeparator` translation key that's `, ` in English. If you remember, it'd be good to come back to this and make use of it here.